spinkit.css

```diff
@@ -2,6 +2,7 @@
 :root {
   --sk-size: 40px;
   --sk-color: #333;
+  --sk-hex-scale: (0.0096153846153846 * var(--sk-size));
```

Comment:
Where does this number come from? Why is it so exact? Can we get rid of if (if we get rid of the parent element)?